dolphinscheduler-scheduler-plugin/dolphinscheduler-scheduler-quartz/src/main/java/org/apache/dolphinscheduler/scheduler/quartz/ProcessScheduleTask.java

```diff
@@ -77,6 +82,14 @@ protected void executeInternal(JobExecutionContext context) {
             return;
         }
 
+        Project project = projectDao.queryProjectById(projectId);
```

Comment:
Just curious, in which scenario will there be a quartz job without project?

Reply:
Directly execute delete sql in db 😢

Reply:
I will close this PR, in DS this will not happen :) unless user directly modifies the db.